pkg/project/controller/controller_test.go

```diff
@@ -38,35 +38,63 @@ func TestSyncNamespaceThatIsTerminating(t *testing.T) {
 		t.Errorf("Unexpected error when handling namespace %v", err)
 	}
 
-	// TODO: we will expect a finalize namespace call after rebase
+	namespaceFinalize := core.NewCreateAction(unversioned.GroupVersionResource{Group: "", Version: "", Resource: "namespaces"}, "", testNamespace)
+	namespaceFinalize.Subresource = "finalize"
+
 	expectedActionSet := []core.Action{
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "buildconfigs"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "policies"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "imagestreams"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "policybindings"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "rolebindings"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "roles"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "routes"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "templates"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "builds"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "namespace"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "deploymentconfig"}, "", kapi.ListOptions{}),
-		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "v1", Resource: "egressnetworkpolicy"}, "", kapi.ListOptions{}),
+		namespaceFinalize,
+		core.NewListAction(unversioned.GroupVersionResource{Group: "", Version: "", Resource: "buildconfigs"}, testNamespace.Name, kapi.ListOptions{}),
```

Comment:
> The old test did not check anything but the total number of actions (i.e. it was useless) so we have not lost anything.

If it was useless, then you needed no changes to pass, right?  I'd sort of expect to see *some* change here even if you didn't have to do all this to have equivalence.